src/cam/ccb.rs

```diff
@@ -22,34 +22,25 @@ impl CCB {
 			panic!("cannot allocate CCB");
 		}
 
-		unsafe {
-			let sizeof_item = mem::size_of_val(&(*ccb).bindgen_union_field[0]);
-			let start = mem::size_of::<bindings::ccb_hdr>() / sizeof_item;
-			let end = mem::size_of::<bindings::ccb>() / sizeof_item;
-			for i in start..end {
-				(*ccb).bindgen_union_field[i] = 0;
-			}
-		}
```

Comment:
I'm not sure it's a good idea to leave CCB un-zeroed. FreeBSD's camcontrol keeps `CCB_CLEAR_ALL_EXCEPT_HDR()` it with (almost?) every allocation ([attrib.c](https://github.com/freebsd/freebsd/blob/f55899f5a9afdebbf970ff95520093f7a780b549/sbin/camcontrol/attrib.c#L134-L141), [fwdownload.c](https://github.com/freebsd/freebsd/blob/bb0c984706c2c72ef11729bd8005eb3d108099b2/sbin/camcontrol/fwdownload.c#L462-L469), [mpt_cam.c](https://github.com/freebsd/freebsd/blob/1d6e4247415d264485ee94b59fdbc12e0c566fd0/usr.sbin/mptutil/mpt_cam.c#L241-L246) to name a few), and [smartmontools outright `bzero()`es it as well](https://github.com/mirror/smartmontools/blob/8e64dd8fef32b00cbe5fd25ad188dce4bb9f52bc/os_freebsd.cpp#L1042-L1068).

Reply:
Hm, let's see.  In camcontrol there are 55 instances of `cam_getccb`.  I see 33 that clear the whole thing and 15 that don't.  So it could go either way.  The problem is that `bindgen_uion_field` no longer exists.  I'll look for another way to clear it.

Reply:
Ok, I restored the old behavior.  Once you're satisfied, I'll squash my commits before you merge.